integration-tests/pipelines/rhtap-cli-e2e.yaml

```diff
@@ -140,6 +140,25 @@ spec:
       params:
         - name: ocp-login-command
           value: "$(tasks.provision-rosa.results.ocp-login-command)"
+    - name: rhtap-e2e-tests
+      when:
+        - input: "$(tasks.test-metadata.results.test-event-type)"
+          operator: in
+          values: ["pull_request"]
+      runAfter:
+        - sprayproxy-provision
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/xinredhat/rhtap-e2e.git
```

Comment:
This should be changed to a team url before commit.

Reply:
the rhtap-e2e task still is in debugging, so this url is expected. When everting is going well, it will be changed to a team url.